src/Illuminate/Testing/TestResponse.php

```diff
@@ -57,23 +65,26 @@ class TestResponse implements ArrayAccess
      * Create a new test response instance.
      *
      * @param  \Illuminate\Http\Response  $response
+     * @param  \Illuminate\Http\Request  $request
      * @return void
      */
-    public function __construct($response)
+    public function __construct($response, $request)
```

Comment:
This is a breaking change.

Reply:
Yes, but it's not used anywhere else in the framework.